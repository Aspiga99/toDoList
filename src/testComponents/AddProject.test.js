import { mount } from '@vue/test-utils'
import Component from '../components/addProject.vue'
import Vue from "vue"

describe("Component", () => {
  it("renders correctly", () => {
    const comp = mount(Component);
    expect(comp.html()).toMatchSnapshot();
  });
});
