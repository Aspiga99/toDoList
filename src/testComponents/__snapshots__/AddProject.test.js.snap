// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component renders correctly 1`] = `
<div id="new-project" style="background: rgb(221, 221, 221);">
  <h3>Add a new Project</h3>
  <form><label>Nome</label> <input type="text" placeholder="Nome" required="required">
    <!----> <label>Descrizione</label> <textarea placeholder="Descrizione" required="required"></textarea>
    <!----> <button id="submit" class="btn btn-primary">Add project</button></form>
  <!---->
</div>
`;
